sskittenapi/catapi/tests/domains/test_cat_domain.py

```diff
@@ -121,3 +121,34 @@ async def test_delete_cat_error_entity_not_found(mock_cat_model_delete_cat: mock
     with pytest.raises(exceptions.CatNotFoundError):
         await cat_domain.delete_cat(cat_id)
     mock_cat_model_delete_cat.assert_called_once_with(cat_id)
+
+
+@pytest.mark.parametrize(
+    "partial_update_cat, cat_id",
+    [
+        (
+            dto.PartialUpdateCat(
+                url="http://placekitten.com/200/300",
+            ),
+            dto.CatID("1"),
+        ),
+        (
+            dto.PartialUpdateCat(
+                url="http://placekitten.com/300/400",
+            ),
+            dto.CatID("2"),
+        ),
+    ],
+)
+@mock.patch("catapi.models.cat_model.partial_update_cat")
+@conftest.async_test
+async def test_partial_update_cat(
+    mock_cat_model_partial_update_cat: mock.Mock,
+    partial_update_cat: dto.PartialUpdateCat,
+    cat_id: dto.CatID,
+) -> None:
+    excepted_cat = dto.Cat
```

Comment:
You are assigning a class to `expected_cat` instead of an instance. 
This should be something like:
```python
expected_cat = dto.Cat(name="Nya-cat", id=dto.CatID("000000000000000000000001"), mtime=now, ctime=now)
```
Or better yet, to also pass this as one of the paramatrized arguments.
You can have multiple ones per test as can be seen in here:

https://github.com/lionbridgeai/panda-api/blob/460a85d6caa012ca69bb27a0105496824a65f3f5/tests/models/test_contributor_assignment_model.py#L360-L370

WDYT?

Reply:
Okay Got it.

Reply:
https://github.com/SShakeeb/Kittenapi/pull/4